evaluator/src/main/kotlin/com/xebia/functional/xef/evaluator/models/errors/SuiteSpecError.kt

```diff
@@ -0,0 +1,58 @@
+package com.xebia.functional.xef.evaluator.models.errors
+
+sealed interface ValidationError {
```

Comment:
Should this file be called `ValidationError` instead?

Reply:
I moved the ValidationError to a new file.